lms/templates/main.html

```diff
@@ -116,11 +100,7 @@
   <meta name="path_prefix" content="${EDX_ROOT_URL}">
   <meta name="google-site-verification" content="_mipQ4AtZQDNmbtOkwehQDOgCxUUV2fb_C0b6wbiRHY" />
 
-  <%include file="${google_analytics_file}" />
-
-% if style_overrides_file:
-  <link rel="stylesheet" type="text/css" href="${static.url(style_overrides_file)}" />
-% endif
+  <%include file="google_analytics.html" />
```

Comment:
Other template names use dashes instead of underscores. Should this be renamed to `google-analytics.html`?